subsys/testsuite/include/zephyr/interrupt_util.h

```diff
@@ -197,6 +197,18 @@ static inline void trigger_irq(int irq)
 	z_mips_enter_irq(irq);
 }
 
+#ifdef CPU_CORTEX_R5
+
+#elif defined(CONFIG_VIM)
```

Comment:
???????? at this point I feel like you are just throwing random code here. Do you think this is acceptable? Did you test it?

Reply:
@carlocaione I apologise for making you feel like that. I never had any intention to throw random code.
I did not test it, I was waiting for the PR tests results and then to run twister locally in my laptop with all the PRs combined (#59191, #61020, #60856). I will test and update the changes.